package.json

```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "chart.js": "^2.8.0",
-    "jquery": "^3.4.1"
+    "jquery": "^3.4.1",
+    "patternomaly": "^1.3.2"
```

Comment:
This is for the patterned charts correct?

Reply:
👍